File: src/main/java/org/shark/example/service/mysql/CreateJsonJoinExampleService.java
package org.shark.example.service.mysql;

import com.google.gson.Gson;
import lombok.RequiredArgsConstructor;
import org.shark.example.dao.repository.JsonJoinExampleRepository;
import org.shark.example.dao.repository.pojo.JsonExampleDo;
import org.shark.example.dao.repository.pojo.JsonJoinExampleDo;
import org.shark.example.service.mysql.pojo.CreateJsonJoinExampleDto;
import org.springframework.stereotype.Service;

@RequiredArgsConstructor
@Service
public class CreateJsonJoinExampleService {

    private final JsonJoinExampleRepository jsonJoinExampleRepository;
    
    public void createJsonExample(CreateJsonJoinExampleDto createJsonJoinExampleDto) {
        JsonJoinExampleDo jsonJoinExampleDo = new JsonJoinExampleDo();
        jsonJoinExampleDo.setJsonExampleId(createJsonJoinExampleDto.getJsonExampleId());
        jsonJoinExampleDo.setDescription(createJsonJoinExampleDto.getDescription());
        jsonJoinExampleRepository.save(jsonJoinExampleDo);
    }
}

File: src/main/kotlin/org/shark/example/service/mysql/CreateJsonJoinExampleService.kt
package org.shark.example.service.mysql

import org.shark.example.dao.repository.JsonJoinExampleRepository
import org.shark.example.dao.repository.pojo.JsonJoinExampleDo
import org.shark.example.service.mysql.pojo.CreateJsonJoinExampleDto
import org.springframework.stereotype.Service

@Service
class CreateJsonJoinExampleService(
    private val jsonJoinExampleRepository: JsonJoinExampleRepository
) {
    fun createJsonExample(createJsonJoinExampleDto: CreateJsonJoinExampleDto) {
        val jsonJoinExampleDo = JsonJoinExampleDo()
        jsonJoinExampleDo.jsonExampleId = createJsonJoinExampleDto.jsonExampleId
        jsonJoinExampleDo.description = createJsonJoinExampleDto.description
        jsonJoinExampleRepository.save(jsonJoinExampleDo)
    }
}
